src/containers/CheckoutPage/CheckoutPage.js

```diff
@@ -298,6 +299,12 @@ export class CheckoutPageComponent extends Component {
           <FormattedMessage id="CheckoutPage.providerStripeAccountMissingError" />
         </p>
       );
+    } else if (isTransactionInitiateBookingTimeNotAvailableError(speculateTransactionError)) {
```

Comment:
`initiateTransactionError` needs some handling too. Otherwise ~parallel checkouts aka different persons pressing of book buttons at the same time might fail miserably.